src/_variables.scss

```diff
@@ -3,103 +3,222 @@
 
-$table-bg-accent: #f5f5f5 !default;
-$table-bg-hover: #f0f0f0 !default;
+// Bootstrap Overrides
+//** Background color used for `.table-striped`.
+$table-bg-accent:         #f5f5f5 !default;
+//** Background color used for `.table-hover`.
+$table-bg-hover:          #f0f0f0 !default;
 
-$btn-default-color: #335075 !default;
-$btn-default-bg: #eaebed !default;
-$btn-default-border: darken($btn-default-bg, 5%) !default;
 
-$btn-primary-bg: $brand-primary !default;
-$btn-primary-border: darken($btn-primary-bg, 20%) !default;
+//== Buttons
+//
+//## For each of buttons, define text, background and border color.
 
-$btn-success-bg: $brand-success !default;
-$btn-success-border: darken($btn-success-bg, 20%) !default;
+// Bootstrap Overrides
+$btn-default-color:       #335075 !default;
+$btn-default-bg:          #eaebed !default;
+$btn-default-border:      darken($btn-default-bg, 5%) !default;
 
-$btn-info-bg: $brand-info !default;
-$btn-info-border: darken($btn-info-bg, 20%) !default;
+$btn-primary-bg:          $brand-primary !default;
+$btn-primary-border:      darken($btn-primary-bg, 20%) !default;
 
-$btn-warning-color: #000 !default;
-$btn-warning-bg: $brand-warning !default;
-$btn-warning-border: darken($btn-warning-bg, 20%) !default;
+$btn-success-bg:          $brand-success !default;
+$btn-success-border:      darken($btn-success-bg, 20%) !default;
 
-$btn-danger-bg: $brand-danger !default;
-$btn-danger-border: darken($btn-danger-bg, 20%) !default;
+$btn-info-bg:             $brand-info !default;
+$btn-info-border:         darken($btn-info-bg, 20%) !default;
 
-$pagination-color: $btn-default-color !default;
-$pagination-bg: $btn-default-bg !default;
-$pagination-border: $btn-default-border !default;
+$btn-warning-color:       #000 !default;
+$btn-warning-bg:          $brand-warning !default;
+$btn-warning-border:      darken($btn-warning-bg, 20%) !default;
+
+$btn-danger-bg:           $brand-danger !default;
+$btn-danger-border:       darken($btn-danger-bg, 20%) !default;
 
-$pagination-hover-color: $btn-default-color !default;
-$pagination-hover-bg: darken($btn-default-bg, 8%) !default;
-$pagination-hover-border: darken($btn-default-border, 12%) !default;
 
-$pager-border-radius: 4px !default;
-
-// Footnotes
-$spacing: .375em !default;
-$ddRtnWidth: 3.375em !default;
-
-// Calendar
-$cal-text-head: #fff !default;
-$cal-text-body: #000 !default;
-$cal-dark: #333 !default;
-$cal-medium: #555 !default;
-$cal-active: #ccc !default;
-$cal-days-link: #111 !default;
-
-//Multimedia Player
-$mm-progress-fg-color: #176ca7 !default;
-$mm-progress-bg-color: #fff !default;
-$mm-cc-fg-color: #fff !default;
-$mm-cc-bg-color: #000 !default;
-$mm-ctrl-fg-color: #fff !default;
-$mm-ctrl-bg-color: #3e3e3e !default;
-$mm-ctrl-outline-color: #4aafff !default;
-$mm-ctrl-cc-color: #4aafff !default;
-$mm-volume-bg-color: #aaa !default;
-$mm-volume-thumb-color: #fff !default;
+//== Media queries breakpoints
```

Comment:
Yes, I can make this change. Is this the only change outstanding? If so, I can do a fix and commit. Otherwise, I'll wait for your comments to come if there are any more. Thanks.